src/annotator/index.js

```diff
@@ -54,15 +54,13 @@ function init() {
     const hypothesisAppsOrigin = new URL(sidebarConfig.sidebarAppUrl).origin;
     portProvider = new PortProvider(hypothesisAppsOrigin);
 
-    sidebar = new Sidebar(
-      document.body,
-      eventBus,
-      portProvider.hostPortFor('sidebar'),
-      guest,
-      sidebarConfig
-    );
+    sidebar = new Sidebar(document.body, eventBus, guest, sidebarConfig);
```

Comment:
An advantage of the previous API is that the caller of `Sidebar` couldn't forget to supply the sidebar app connection. On the other hand, we need something like `onFrameConnected` for guest frames later.

Reply:
Yes, I agree it had some advantages, but made it a little asymmetric.